src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -137,6 +137,13 @@ bool is_absolute_file_path(const std::string& path);
  */
 void create_directory_recursive(const std::string& path);
```

Comment:
do we still need this overloads for string and wstring?

Reply:
I didn't focus on utils remove in this PR, as this is internal function and can be removed any time.
For `std::string` I think it can be removed.
for `wstring` it has to be checked as on Windows and unicode paths such function can be usful to make correct conversion to path, or use user mast create it correctly.